pkg/security/admission/byrestrictions.go

```diff
@@ -50,3 +50,13 @@ func pointValue(constraint *kapi.SecurityContextConstraints) int {
 	}
 	return points
 }
+
+// allowsHostPathVolume checks to see if the FSTypeHostPath is in the Volumes slice.
+func allowsHostPathVolume(scc *kapi.SecurityContextConstraints) bool {
```

Comment:
probably any volume type other than emptydir/secret/configmap/downwardapi is worth bumping priority